xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -700,6 +702,50 @@ namespace PVR
                                             CVariant{timer->Title()});
   }
 
+  class AsyncRecordingAction : private IRunnable
+  {
+  public:
+    bool Execute(const CFileItemPtr &item = CFileItemPtr());
```

Comment:
Don't take a shared_ptr by reference, should be by value to ensure ref count is correctly incremented